Exercise_B/index.html

```diff
@@ -18,17 +18,60 @@
   </head>
 
   <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
+      <!--Header (flower shop logo block element)-->
+    <nav class="navbar bg-light">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="#">
+          <img src="./assets/images/logo.png" alt="logo" height="60">
+        </a>
+      </div>
+    </nav>
+
+        <!--Banner Jumbotron of the image-->
+      <div class="container-fluid px-0 imgContainer">

```

Comment:
For this container, i would suggest you change it as the items are stretched from left to right and should be in a row format you can make use of Flex or Grid 
You can look at the examples here [https://getbootstrap.com/docs/5.2/layout/grid/](url)